### ISSUE TITLE
Make game system automatically detected for movie userfiles

### ISSUE THREAD

**Issue by ThunderAxe31** [TASVideos/tasvideos#1029]
Just as it used to be on old site. This would avoid incorrect timings displayed. Related: #1013

**Comment by adelikat** [comment]
the system parsed from the userfile happens server side so it happens after the user would pick the system.  Should it override what the user picked?  What about situations like .gmv and 32x where the parser would override the correct 32x selection with an incorrect GENS code?

**Comment by vadosnaprimer** [comment]
What about "autodetect" as a default option?

**Comment by adelikat** [comment]
Yeah, that's probably a convenient way to handle it, and with autodetect defaulting to true
